manifests/php.pp

```diff
@@ -67,15 +61,46 @@
     $env = delete($env_temp, nil)
 
     $install_option = $ensure ? { absent => 'purge', default => 'install' }
-    exec { 'newrelic-install':
-        environment => $env,
-        command => "newrelic-install ${install_option}",
-        path => ['/bin', '/usr/bin'],
-        require => Package['newrelic-php5'],
-        refreshonly => true
+
+
+
+    if ($ensure == absent) {
+      Exec['newrelic-install'] -> Package['newrelic-php5']
```

Comment:
If we're ensuring the package is absent, make sure that the `purge` Exec runs before the package is removed.